schedule/yast/addon_extensions_http_ftp/addon_extensions_http_ftp.yaml

```diff
@@ -13,7 +12,9 @@ schedule:
   - installation/licensing/accept_license
   - installation/registration/register_via_scc
   - installation/module_selection/skip_module_selection
-  - installation/addon_products_sle
+  - installation/add_on_product/add_add_on_http_repo_url
+  - installation/add_on_product_installation/add_add_on_ftp_repo_url
```

Comment:
Why aren't these two modules in the same directory (`add_on_product_installation`)?

Reply:
they are in different screens, you can check the video, it will depend if it is is already some addon added.